include/deal.II/base/mpi_remote_point_evaluation.h

```diff
@@ -108,6 +108,16 @@ namespace Utilities
         std::vector<Point<dim>> reference_point_values;
       };
 
```

Comment:
Can you move the implementation to the cc file?

Reply:
Also, this seems a small, self contained change. Can you make a separate PR for this one?